app/src/main/java/io/github/jbellis/brokk/gui/InstructionsPanel.java

```diff
@@ -777,12 +776,6 @@ private void triggerContextSuggestion(@Nullable ActionEvent e) { // ActionEvent
     private void processInputSuggestions(long myGen, String snapshot) {
         logger.trace("Starting suggestion task generation {}", myGen);
 
-        if (contextManager == null) {
-            logger.warn("Task {} cannot provide suggestions: ContextManager is not available.", myGen);
-            SwingUtilities.invokeLater(() -> showFailureLabel("Context features unavailable"));
-            return;
-        }
```

Comment:
`@Nullable` recently removed from `contextManager` field, so fixed this and other `RedundantNullCheck` warnings in `InstructionsPanel`.